app/renderer/components/config_menu.js

```diff
@@ -39,22 +46,42 @@ module.exports = {
     </div>
     `,
 
-    methods: {
-        clearTasks() {
-            TaskConfig.clearTasks();
-            AppStatus.activeSuite = 0;
-        },
-        resetConfig() {
-            this.config.bottomBar=true;
-            this.config.animatedSpinner=true;
-        },
-        onClose() {
-            this.config.bottomBar = AppStatus.config.bottomBar;
-            this.config.animatedSpinner = AppStatus.config.animatedSpinner;
-        },
-        onSave() {
-            AppStatus.config.bottomBar = this.config.bottomBar;
-            AppStatus.config.animatedSpinner = this.config.animatedSpinner;
-        }
+  methods: {
+    exportTasks(){
+      let content = TaskConfig.getData() ;
+      content = {
+        "suites":content,
+        "version":"0.3.4"
+      };
+      content = JSON.stringify(content) ;
+      dialog.showSaveDialog(
+        {
+          defaultPath: require('os').homedir() + "/gtask.json",
+          filters: [
+            { name: 'json', extensions: ['json'] }
+          ]}, (filename)=> {
+            if(filename){
+              fs.writeFile(filename, content, (err) => {
```

Comment:
aaand what happens if no filename? and why wouldn't be filename?

If there is a chance that the filename doesn't exists (i guess it is because the user didn't set a path or something) it doesn't makes sense all the processing we did to create the content.

I would say it is better to create the content just before writing it. Unless there is a reason not to do it that way

Reply:
Actually, I would say that the user could not save the file without select a filename, but in some guides I read people check it so I did... maybe for something we're ommiting... so, I think we should let this check and just modify when the content is created as you said :D